plugins/source/azure/docs/tables/azure_cdn_custom_domains.md

```diff
@@ -15,15 +15,13 @@ This table depends on [azure_cdn_endpoints](azure_cdn_endpoints.md).
 |_cq_sync_time|Timestamp|
 |_cq_id|UUID|
 |_cq_parent_id|UUID|
-|subscription_id|String|
```

Comment:
Was `subscription_id` removed? I can't find it in the new table

Reply:
done